UNITTESTS/mbed_unittest.py

```diff
@@ -71,9 +71,8 @@ def _mbed_unittest_test(options, cwd, pwd):
                                  clean=options.clean)
 
     if options.compile_only:
-        # Create makefiles
-        src_path = os.path.relpath(pwd, options.build)
-        tool.create_makefiles(path_to_src=src_path,
+        # Create makefiles        
+        tool.create_makefiles(path_to_src=cwd,
```

Comment:
As we're now using "modern CMake" for both Mbed CLI 1 & Mbed CLI 2, we can safely delete the old-fashion `unittest.cmake`s from the entire Mbed OS:
```bash
find . -name "unittest.cmake" | xargs rm
```
I've tested it.

Reply:
I agree, we can safely remove these - I think these were removed in the original PR before we started this second attempt.

Reply:
Removing the unittest.cmake in the separate PR https://github.com/ARMmbed/mbed-os/pull/14547